features/topics.feature

```diff
@@ -58,22 +43,17 @@ Scenario: Choosing and ordering lead organisations within a topic
     |Ministry of War|
 
 Scenario: Viewing the list of topics
-  Given the topic "Higher Education" contains some policies
-  And the topic "Science and Innovation" contains some policies
+  Given a topic called "Higher Education" exists
+  And a topic called "Science and Innovation" exists
   When I visit the list of topics
   Then I should see the topics "Higher Education" and "Science and Innovation"
 
 Scenario: Visiting a topic page
-  Given the topic "Higher Education" contains some policies
+  Given a topic called "Higher Education" exists
+  And a topic called "Science and Innovation" exists
   And the topic "Higher Education" is related to the topic "Scientific Research"
   When I visit the "Higher Education" topic
-  Then I should see published policies belonging to the "Higher Education" topic
-  And I should see a link to the related topic "Scientific Research"
-
-Scenario: Featuring content on a topic page
```

Comment:
No point losing a random bit of test coverage. Better to remove everything as and when the functionality is finally removed.